From c62cfb4b929686d7bf2c9854d00677c2e9448648 Mon Sep 17 00:00:00 2001
From: Rob Herring <rob.herring@calxeda.com>
Date: Tue, 3 Jan 2012 16:57:33 -0600
Subject: ARM: shmobile: remove NR_IRQS

Remove NR_IRQS and explicitly include mach/irqs.h as needed. shmobile
properly allocates irq_descs for each irqchip, so setting .nr_irqs for
each machine is not needed.

Signed-off-by: Rob Herring <rob.herring@calxeda.com>
Signed-off-by: Simon Horman <horms@verge.net.au>
(cherry picked from commit 250a27237e0e0694f8f8451594a0f03e0a4f5a73)

Conflicts:

	arch/arm/mach-shmobile/board-ag5evm.c
	arch/arm/mach-shmobile/board-bonito.c
	arch/arm/mach-shmobile/board-g3evm.c
	arch/arm/mach-shmobile/board-g4evm.c
	arch/arm/mach-shmobile/board-kota2.c
	arch/arm/mach-shmobile/board-mackerel.c
	arch/arm/mach-shmobile/board-marzen.c
	arch/arm/mach-shmobile/include/mach/irqs.h
	arch/arm/mach-shmobile/intc-r8a7740.c
	arch/arm/mach-shmobile/intc-sh7367.c
	arch/arm/mach-shmobile/intc-sh7372.c
	arch/arm/mach-shmobile/intc-sh7377.c
	arch/arm/mach-shmobile/setup-r8a7740.c
	arch/arm/mach-shmobile/setup-r8a7779.c
	arch/arm/mach-shmobile/setup-sh7367.c
	arch/arm/mach-shmobile/setup-sh7372.c
	arch/arm/mach-shmobile/setup-sh7377.c
	arch/arm/mach-shmobile/setup-sh73a0.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 arch/arm/mach-shmobile/setup-sh73a0.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-shmobile/setup-sh73a0.c b/arch/arm/mach-shmobile/setup-sh73a0.c
index 20e71e5..7bf740e 100644
--- a/arch/arm/mach-shmobile/setup-sh73a0.c
+++ b/arch/arm/mach-shmobile/setup-sh73a0.c
@@ -31,6 +31,7 @@
 #include <linux/sh_intc.h>
 #include <linux/sh_timer.h>
 #include <mach/hardware.h>
+#include <mach/irqs.h>
 #include <mach/sh73a0.h>
 #include <asm/mach-types.h>
 #include <asm/mach/arch.h>
-- 
1.7.10.2.565.gbd578b5

